packages/elpi/elpi.1.15.0/opam

```diff
@@ -14,7 +14,7 @@ build: [
 ]
 
 depends: [
-  "ocaml" {>= "4.07.0" }
+  "ocaml" {>= "4.07.0" & <= "4.15.0" }
```

Comment:
Why <= 4.15?

Reply:
Arrgh, I think "I made a mistake".  *grin*  I wanted to exclude all ocaml 5.X versions.  Let me try again.  This time, I'll use
` < "4.15.0"`.  Though I wonder if `< "5"` would work.

Reply:
This is strange, the original bound should have worked